polaris-tokens/tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2020",
+    "lib": ["es2020"],
+    "outDir": "out",
```

```suggestion
    "outDir": "dist",
```

This was briefly discussed in @samrose3's PR, to configure `tsc` to run from the root, but we should probably settle on an `outDir` naming convention for new libs.

With that said, @samrose3 is there any other `compilerOptions` that need to be set here? If so, we should add a doc about what options to set for libs that do and don't bundle with `tsc`.